### DISCUSSION TITLE
火狐-英文字母上边缘伪影

### DISCUSSION THREAD

**Discussion by Yinchieh** [F9y4ng/GreasyFork-Scripts#397] [问答专区(Question / Answer)] [answered]
浏览器为火狐，脚本为2024.11.02.1版本。
### 使用Futura字体，英文字母上方会出现如下伪影。
![image](https://github.com/user-attachments/assets/8c0ea129-fb6f-4ffc-9d0e-f3298ae11d16)
![image](https://github.com/user-attachments/assets/b855829b-c68f-4aca-a910-2c0d6394f515)

### 换一种字体会好点：
![image](https://github.com/user-attachments/assets/dbc80122-1ed3-4e57-8e06-2fcc4e3a2a98)

**Comment by F9y4ng** [comment, marked as answer]
是的，这种字体造成的渲染错误，不仅仅会出现在Firefox上，在Chromium系浏览器上也存在。比如 低分辨率下 `微软雅黑、XHei` 等字体开启描边后会出现字体渲染错误，而其他字体正常。根源在windows本身的渲染问题上，所以，换另外一个字型相似的字体使用是解决之道。

不过，如果你正在使用Firefox>126版本，可以尝试打开字体比例缩放功能，通过字体缩放来解决这个渲染问题。

**Reply by Yinchieh** [reply]
谢谢，我也试了把字体描边尺寸和阴影尺寸都关掉也会好一点。
